tests/containers/docker_image.pm

```diff
@@ -37,13 +38,16 @@ sub run {
     scc_apply_docker_image_credentials() if (get_var('SCC_DOCKER_IMAGE'));
 
     for my $iname (@{$image_names}) {
+        record_info 'image name', '$iname';
         test_container_image(image => $iname, runtime => $runtime);
         build_container_image(image => $iname, runtime => $runtime);
-        if (check_os_release('suse', 'PRETTY_NAME')) {
+
+        if (check_os_release('suse', 'PRETTY_NAME') && is_sle_container_supported) {
```

Comment:
so again just trying to understand things , if I will replace function call with content of it we will get something like : 

```
if (check_os_release('suse', 'PRETTY_NAME') ) {
   unless (check_os_release('opensuse', 'PRETTY_NAME') && is_sle()) {
```
sorry but I can not understand what this check suppose to say . maybe again I am missing that some functions called to check host OS and others container ?

Reply:
`check_os_release('suse', 'PRETTY_NAME')` is true for all the suse distros. think of `is_sle_container_supported` as a *filter* for no-supported distros.

Reply:
I think you missing my point . 
1. first you calling `check_os_release('suse','PRETTY_NAME')` and getting **true** 
2. than you calling `check_os_release('opensuse','PRETTY_NAME')` which does not make sense because you already **know** at this point that it will return you false 

so for me it is like having something like this : 
```
x = 10 ; 
if ( x > 5 ) {
   unless( x == 0 ) { 
      print "x greater than 5 and not equal 0"
```
this code will work the same with and without `unless (x == 0 )`

Reply:
i am resolving this. see latest commit for another approach